README.md

```diff
@@ -1,28 +1,147 @@
-# Kubernetes cluster-api-provider-vsphere Project
+# Kubernetes Cluster API Provider vSphere
 
-This repository hosts a concrete implementation of a provider for vsphere for the [cluster-api project](https://github.com/kubernetes-sigs/cluster-api).
+[![Go Report Card](https://goreportcard.com/badge/github.com/kubernetes-sigs/cluster-api-provider-vsphere)](https://goreportcard.com/report/github.com/kubernetes-sigs/cluster-api-provider-vsphere)
 
-## Community, discussion, contribution, and support
+Kubernetes-native declarative infrastructure for vSphere.
 
-Learn how to engage with the Kubernetes community on the [community page](http://kubernetes.io/community/).
+## What is the Cluster API Provider vSphere
 
-* Join our Cluster API vSphere Provider sessions
-  * Bi-weekly on Mondays @ 1:00 PT on [Zoom](https://zoom.us/j/875399243) starting on 8/13
-  * Previous meetings: \[ [notes](https://docs.google.com/document/d/1jQrQiOW75uWraPk4b_LWtCTHwT7EZwrWWwMdxeWOEvk/edit?usp=sharing) \]
+The [Cluster API][cluster_api] brings
+declarative, Kubernetes-style APIs to cluster creation, configuration and
+management. Cluster API Provider vSphere is a concrete implementation of
+Cluster API for vSphere.
```

Comment:
Hi @ykakarap,

Please change the end of the sentence to be:

Cluster API for vSphere (CAPV).